pkg/registry/core/service/storage/storage_test.go

```diff
@@ -5832,4 +5834,979 @@ func TestDeleteDryRun(t *testing.T) {
 	}
 }
 
-//FIXME: Tests for update()
+type svcTestCase struct {
+	svc         *api.Service
+	expectError bool
+
+	// We could divine these by looking at the Service, but that's a
```

Comment:
divine? as in identify?

Reply:
meaning "figure out" or "discover".  Will reword.